doc/source/gotchas.rst

```diff
@@ -1,18 +1,92 @@
 .. currentmodule:: pandas
 .. _gotchas:
 
+********************************
+Frequently Asked Questions (FAQ)
```

Comment:
I'm not sure it really makes sense to call this new page "FAQ" until we rephrase the section headers as questions.